configuration/blockly/rules-blockly-standard-ext.md

```diff
@@ -92,6 +112,13 @@ A color of r=255, g=255, b=255 would therefore result into #ffffff preceded by a
 Note that typically RGB values are not used within openHAB for color states but instead the Color-Type expects a HSB Type.
 See the special hsb-conversion block below.
 
+- pick a color to return the value as an RGB type
+- chose a random color as RGB type
+- define an RGB color based on red, green and blue.
+- blend two colors with a ratio - this block can be used to dynamically transition from one color to the other by applying the ratio as a computed value in a rule.
+- calculate the HSB value of the RGB value
```

Comment:
BTW: This block could be refactored to use ColorUtil in openHAB Core, see the `CoreUtil` actions: https://github.com/openhab/openhab-core/blob/36cafd765f6f747cb1e26a021fae7c25539e40c3/bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/CoreUtil.java#L25.

Reply:
Good point, not sure if that was available back then. 
Would that be actions.CoreUtil.hsbTosRgb? 
However, it is bit more complicated than only using it because the input is a hexvalue (Blockly generates hex values), so we still need to have some extra code that first extracts r / g /b and then yes, I could use the ColorUtil class.

I will do work on that in the future.